lib/src/migrators/module/references.dart

```diff
@@ -361,10 +368,15 @@ class _ReferenceVisitor extends RecursiveAstVisitor {
 
   /// Given a URL from a `@use` or `@forward` rule, loads and visits the
   /// stylesheet it points to and returns its canonical URL.
-  Uri _loadUseOrForward(Uri ruleUrl) {
+  Uri _loadUseOrForward(Uri ruleUrl, AstNode nodeForSpan) {
```

Comment:
Is there any reason that this needs to be `AstNode` and not `SassNode`?

Reply:
Just consistency with the convention from the Sass codebase.